lib/manifest.js

```diff
@@ -290,6 +290,18 @@ internals.manifest = [
         list: true,
         useFilename: internals.passthruOn('name')
     },
+    {
+        place: 'events',
+        method: 'event',
+        signature: ['events'],
+        list: true
+    },
+    {
+        place: 'event-listeners',
+        method: 'on',
+        signature: ['criteria', 'listener'],
+        list: true
+    },
```

Comment:
`event-listeners` should have `after: ['events']` (or whatever we rename them).  If the calls were to happen out of order, hapi would complain: `srv.on('my-event', listener); srv.event('my-event');`.